queries.sql

```diff
@@ -1,3 +1,51 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+
+
+-- List the name of all animals born between 2016 and 2019.
+
+SELECT *
+FROM animals
+WHERE date_of_birth > '2016-01-01' and date_of_birth < '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+
+SELECT *
+FROM animals
+WHERE neutered = true and escape_attempts < 3;
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+
+SELECT name, date_of_birth
+FROM animals
+WHERE name = 'Agumon' or name ='Pikachu';
```

Comment:
- [ ] Kindly make sure to select only the date_of_birth here, the names are not required